src/components/AccessControlNew.tsx

```diff
@@ -73,12 +72,14 @@ const AccessControlNew = () => {
         return {
             ID: formRule.ID,
             Name: formRule.Name,
+            Description: formRule.Description,
             Source,
             Destination,
             sourcesNoId,
             destinationsNoId,
             groupsToSave,
-            Flow: formRule.Flow
+            Flow: formRule.Flow,
+            Disabled: formRule.Disabled
```

Comment:
Codacy has a fix for the issue: [Replace `············Disabled:·formRule.Disabled` with `······Disabled:·formRule.Disabled,`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      Disabled: formRule.Disabled,
```